sample-generation/src/main/java/no/unit/nva/sampleproject/PublicationGenerator.java

```diff
@@ -0,0 +1,27 @@
+package no.unit.nva.sampleproject;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import java.util.List;
+import java.util.Map;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.model.Organization;
+import no.unit.nva.model.Publication;
+
+public class PublicationGenerator {
+
+    public static Publication createPublication() {
+        return new Publication.Builder()
```

Comment:
A small class for creating a random publication. This will be extended to create a complete list of all possible Resource types with all possible Publication Instances and Contexts